### DISCUSSION TITLE
Obtained a new access token.

### DISCUSSION THREAD

**Discussion by mamoebius** [bisand/node-red-contrib-panasonic-comfort-cloud#15] [General]
Hello,
 
through your API I control two air conditioners. I read the values of the system and reset div. values.
For example: set temperature, etc. This has worked well until 15.4.23. From this date on, any request or even set is met with a " Obtained a new access token." qittiert. Thus after some time my account is spert with the message; "too mach reqests".
Why is now always a new token created?
 
Mit freundlichen Grüßen Möbius

Antwort von Panasonic:

Sehr geehrter Herr Möbius,

vom europäischen IOT-Team habe ich folgende Rückmeldung bekommen.
Das Nutzen einer derartigen Software wird von uns nicht unterstützt.
Wenn nach in Kraft treten der Sperrung 48h kein Verbindungsversuch unternommen wurde, wird die PANASONIC-ID wieder freigeschaltet.
Das nutzen der Software führt sehr wahrscheinlich erneut zu einer Sperrung der ID führen.
Wir bitten um Kenntnisnahme.

Mit freundlichen Grüßen, Best regards, | 敬具

Marc-André Freudenberger
Product Specialist CAC
Panasonic Heating & Cooling Solutions Europe
Email: [Marc-Andre.Freudenberger@eu.panasonic.com](mailto:Marc-Andre.Freudenberger@eu.panasonic.com)

**Comment by bisand** [comment]
It seems like Panasonic has changed something that requires changes in this component. I will do some testing to see if I can reproduce the problem and hopefully fix it. Feel free to submit a pull request if you see a quick fix.

Thanks!

**Comment by mamoebius** [comment]
Hello,is there already something new because of the new token on each query or send changes? i have helped myself so far by working with different accounts in a flow and extremely extended the query time. Using an account every 30min does not get me a lock. But with this I can not run the system as planned, it is too sluggish.I am glad to hear positive answers.MfG Möbius MarioAnfang der weitergeleiteten Nachricht:Von: André Syvertsen Biseth ***@***.***>Datum: 27. April 2023 um 07:56:25 MESZAn: bisand/node-red-contrib-panasonic-comfort-cloud ***@***.***>Kopie: mamoebius ***@***.***>, Author ***@***.***>Betreff: Aw: [bisand/node-red-contrib-panasonic-comfort-cloud] Obtained a new access token. (Discussion #15)Antwort an: bisand/node-red-contrib-panasonic-comfort-cloud ***@***.***>﻿It seems like Panasonic has changed something that requires changes in this component. I will do some testing to see if I can reproduce the problem and hopefully fix it. Feel free to submit a pull request if you see a quick fix.Thanks!—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by bisand** [comment]
Is this still an issue?
